Kentico.Kontent.Management/IManagementClient.cs

```diff
@@ -513,5 +515,47 @@ public interface IManagementClient
         /// <param name="user">Represents an user that is to be modified.</param>
         /// <returns>Returns the modified user.</returns>
         Task<UserModel> ModifyUsersRolesAsync(UserIdentifier identifier, UserModel user);
+
+        /// <summary>
+        /// Returns strongly typed listing of projects.
+        /// The Content management API returns a dynamically paginated listing response limited to up to 100 objects.
+        /// To check if the next page is available use <see cref="IListingResponseModel{T}.HasNextPage"/>.
+        /// For getting next page use <see cref="IListingResponseModel{T}.GetNextPage"/>.
+        /// </summary>
+        /// <returns>The <see cref="IListingResponseModel{SubscriptionProjectModel}"/> instance that represents the listing of projects.</returns>
+        Task<IListingResponseModel<SubscriptionProjectModel>> ListSubscriptionProjectsAsync();
```

Comment:
Do we want to remark extra exception with insufficient API key?

Reply:
Let's apply the rule - if the exception is raised based on SDK code - have it in remarks, otherwise rely on exception text and code.